### PR TITLE
+ [ios] add gcanvas demo & fixbug wxdevtool crash & move pod spec to root directory

### PR DESCRIPTION
**Pull request by kfeagle**
+ [ios] add gcanvas demo
+ [ios] fixbug wxdevtool crash
+ [ios] move pod spec to root directory
+ [ios] change input keyborad type tip

### CODE REVIEW

File: ios/playground/Podfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ def common
     pod 'SDWebImage', '3.7.5'
     pod 'SocketRocket', '0.4.2'
     pod 'ATSDK-Weex', '0.0.1'
+    
+    pod 'WeexGcanvas', '0.0.1.5'
```
Comment (acton393) [inline ios/playground/Podfile]: more notes for WeexGcanvas

File: WeexSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  s.authors      = { "cxfeng1"      => "cxfeng1@gmail.com",
+                     "boboning"     => "ningli928@163.com",
+                     "yangshengtao" => "yangshengtao1314@163.com",
+                     "kfeagle"      => "sunjjbobo@163.com",
+                     "acton393"     => "zhangxing610321@gmail.com"
+                   }
+  s.platform     = :ios
+  s.ios.deployment_target = '7.0'
+  s.source =  { :path => '.' }
+  s.source_files = 'ios/playground/WeexSDK/Sources/**/*.{h,m,mm,c}'
```
Comment (acton393) [inline WeexSDK.podspec]: there is something wrong with source_files path

Comment (kfeagle) [inline WeexSDK.podspec]: Yes, you're right , thanks for your reminder
I fix it

File: ios/playground/Podfile (hunk 1 above)
Comment (kfeagle) [inline ios/playground/Podfile]: I add as follow,is it ok?
# WeexGcanvas is added by Weex Plugin, more info at https://market.dotwe.org/ext/list.htm

**Pull request merged**